packages/container/libraries/container/src/binding/models/BindingFluentSyntax.spec-d.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  assertType,
+  beforeAll,
+  describe,
+  expectTypeOf,
+  it,
+  Mocked,
+  vitest,
+} from 'vitest';
+
+import { Newable, ServiceIdentifier } from '@inversifyjs/common';
+
+import {
+  BindInWhenOnFluentSyntax,
+  BindToFluentSyntax,
+} from './BindingFluentSyntax';
+
+describe('BindToFluentSyntax', () => {
+  let bindToFluentSyntaxMock: Mocked<BindToFluentSyntax<unknown>>;
+
+  beforeAll(() => {
+    bindToFluentSyntaxMock = {
+      toResolvedValue: vitest.fn() as unknown,
+    } as Partial<Mocked<BindToFluentSyntax<unknown>>> as Mocked<
+      BindToFluentSyntax<unknown>
+    >;
+  });
+
+  describe('.toResolvedValue', () => {
```

Comment:
Some tests are missing, we need to test `ResolvedValueMetadataInjectOptions` with `isOptional` and `isMultiple`. Spoiler alert: we need to update typings to fix that use case